source/unified-test-format/unified-test-format.rst

```diff
@@ -1398,6 +1398,14 @@ The structure of each object is as follows:
   message counts are the same and that there are no extra messages emitted by the
   client during the test run.
 
+- ``ignoreLogMessages``: Optional array of `expectedLogMessage`_ objects. List of
+  messages, which should be ignored on the corresponding client while executing `operations`_.
```

Comment:
`ignoreExtraLogs` is pretty clear, but I think `ignoreLogMessages` can still use some clarification.

I'm assuming the following:

 * The array is an unordered set of messages (unlike `messages` where order is significant)
 * When a log message is observed, the test runner will first check if it matches anything in `ignoreLogMessages` (following the same "exact match" rules as `messages`). If so, the log message is discarded _before_ being considered for `messages` evaluation.

If that's correct, can you capture this in the spec?

----

Also, the language in `messages` may need some revision to avoid conflict with the new fields being introduced. I'd suggest adding the following paragraph to the end of `messages`:

```
  Note: ``ignoreMessages`` and ``ignoreExtraMessages`` may exclude log messages
  from this evaluation.
```

And lastly, I'd suggest renaming the field names:

 * `ignoreLogMessages` -> `ignoreMessages`
 * `ignoreExtraLogs` -> `ignoreExtraMessages`

I think these are more consistent since the original field is just `messages`. "Log" is already included in the higher-level `expectLogMessages` field name, so we already have that context. `ignoreExtraMessages` would also be consistent with `ignoreExtraEvents` (where the main array field there is `events`, like we have with `messages` here).

Reply:
Thanks for the good suggestions, done.